src/domain/community/contributor/contributor.base.entity.ts

```diff
@@ -3,19 +3,11 @@ import { Column, JoinColumn, OneToOne } from 'typeorm';
 import { Agent } from '@domain/agent/agent/agent.entity';
 import { IContributorBase } from './contributor.base.interface';
 import { NameableEntity } from '@domain/common/entity/nameable-entity/nameable.entity';
-import { NAMEID_MAX_LENGTH_SCHEMA } from '@common/constants/entity.field.length.constants';
 
 export class ContributorBase
   extends NameableEntity
   implements IContributorBase
 {
-  @Column('varchar', {
-    length: NAMEID_MAX_LENGTH_SCHEMA,
-    nullable: false,
-    unique: true,
-  })
-  nameID!: string;
```

Comment:
this overwrite has to remain - it overwrites the uniqueness of nameIDs of contributors.